test/GoodGhosting_Funds_Redeemed_From_Contract_On_EarlyWithdraw.test.js

```diff
@@ -198,9 +199,15 @@ contract("GoodGhosting_Funds_Redeemed_On_Early_Withdraw", (accounts) => {
             for (let i = 2; i < players.length - 1; i++) {
                 const player = players[i];
                 const result = await goodGhosting.withdraw({ from: player });
-                truffleAssert.eventEmitted(result, "Withdrawal", (ev) => {
+                truffleAssert.eventEmitted(result, "Withdrawal", async (ev) => {
                     console.log(`player${i} withdraw amount: ${ev.amount.toString()}`);
-                    return ev.player === player;
+                    if (GoodGhostingArtifact === GoodGhostingPolygon) {
+                        rewardToken = new web3.eth.Contract(daiABI, providersConfigs.wmatic);
```

Comment:
please check submitted changes to `test/GoodGhosting_Attacker_Sends_DAI_Externally.test.js`.
`rewardToken` could be initialized just once in the initialization function.

Reply:
oh my bad actually I thought it was failing for mainnet initialization I got a few errors while mainnet testing